packages/styled-system/package.json

```diff
@@ -45,7 +45,8 @@
   },
   "sideEffects": false,
   "scripts": {
-    "build": "tsup src/index.ts --dts",
+    "build": "tsup src/index.ts && pnpm build:dts",
+    "build:dts": "tsup --entry src/index.ts --entry src/theming.types.ts --entry src/shared.types.ts --dts-only",
     "dev": "pnpm build -- --watch",
```

Comment:
In that case, let's restructure this a bit so that the `dev` command and `build:fast` can work.

Reply:
Fixed 4a2d4531bdd2d42a4c1f13fbff42dc120a53e8c1